Makefile

```diff
@@ -67,7 +67,12 @@ git-blame:
 .git/hooks/pre-commit: scripts/pre-commit
 	./scripts/setup-pre-commit-hook
 
-dev-env: $(PELORUS_VENV) exporters git-blame .git/hooks/pre-commit
+.PHONY: cli_dev_tools
+cli_dev_tools:
+	./scripts/install_dev_tools.sh -v $(PELORUS_VENV)
+
+dev-env: $(PELORUS_VENV) cli_dev_tools exporters git-blame \
+         .git/hooks/pre-commit
```

Comment:
Can we make this use file targets instead of a phony target? That will mean the script won't need to run every time. I know it will skip installing if it's present, but why not skip it in the first place?

Reply:
Added as well, but I think having this in a file is also nice to have in case I want to re-download only one tool e.g. newest and latest oc client then:
```
rm .venv/bin/oc
scripts/install_dev_tools
```

Reply:
Oh, my mistake! I thought the other targets depended upon `dev-env`, not `$(PELORUS_VENV)`. The way you had it would be fine.

The way I was thinking would be a target per tool, and the script could take an argument specifying which tool to download.